docs/api/batch.md

```diff
@@ -6,8 +6,8 @@ The Batch API is used to request the ability to transfer LFS objects with the
 LFS server. The Batch URL is built by adding `/objects/batch` to the LFS server
 URL.
 
-Git remote: https://git-server.com/foo/bar  
-LFS server: https://git-server.com/foo/bar.git/info/lfs  
+Git remote: https://git-server.com/foo/bar
+LFS server: https://git-server.com/foo/bar.git/info/lfs
```

Comment:
Ah, I remember thinking that I needed to not stage those changes and then totally forgot. I think we should go with the `<br>`, since I think it's fairly common for editors to trim trailing whitespace off the end of a line.

I went with that in 2a1e63e, but was thinking about adding markdown tables the whole time.